new-site/src/docs/elements/components/buttons/index.mdx

```diff
@@ -20,26 +17,18 @@ import Usage from './usage.mdx';
 </StatusLabel>
 
 <LeadParagraph>
-Buttons are meant to make actions easily visible and understandable to the user. HDS offers different kinds of button variations which each suit for different needs.
+  Buttons are meant to make actions easily visible and understandable to the user. HDS offers different kinds of button
+  variations which each suit for different needs.
 </LeadParagraph>
 
-<Tabs>
-  <Tabs.TabList className="pageTabsList">
-    <Tabs.Tab>Usage</Tabs.Tab>
-    <Tabs.Tab>Code</Tabs.Tab>
-    <Tabs.Tab>Accessibility</Tabs.Tab>
-    <Tabs.Tab>Customisation</Tabs.Tab>
-  </Tabs.TabList>
-  <Tabs.TabPanel>
-    <Usage />
-  </Tabs.TabPanel>
-  <Tabs.TabPanel>
-    <Code />
-  </Tabs.TabPanel>
-  <Tabs.TabPanel>
-    <Accessibility />
-  </Tabs.TabPanel>
-  <Tabs.TabPanel>
-    <Customisation />
-  </Tabs.TabPanel>
-</Tabs>
+<PageTabs pageContext={props.pageContext}>
+  <PageTabs.TabList>
+    <PageTabs.Tab href="/usage">Usage</PageTabs.Tab>
+    <PageTabs.Tab href="/code">Code</PageTabs.Tab>
+    <PageTabs.Tab href="/accessibility">Accessibility</PageTabs.Tab>
+    <PageTabs.Tab href="/customisation">Customisation</PageTabs.Tab>
+  </PageTabs.TabList>
+  <PageTabs.TabPanel>{props.children}</PageTabs.TabPanel>
+</PageTabs>
+
+export default ({ pageContext, children }) => <Layout pageContext={pageContext}>{children}</Layout>;
```

Comment:
Is this layout duplicate?

Reply:
No but it can be self-closing tag

Reply:
Actually no, it takes the children as property